### ISSUE TITLE
No Local Install Instructions

### ISSUE THREAD

**Issue by m4v15** [FACK1/GBMQ#59]
Could you please add information on how to run this locally:

- Technologies required (Node, Postgres if using a local database)
- Environment variables - names, what to call the file, where to put the file

**Comment by m4v15** [comment]
Instructions on setting up the database are slightly less important, but you could have something short when you talk about whatever DB environment variables you need

**Comment by m4v15** [comment]
Also:
- what scripts to run for start, dev, client dev?

**Comment by mhmdtshref** [comment]
Will add it by tomorrow, is that good ?

**Comment by m4v15** [comment]
Yeah, whenever works for you, I'll just raise issues and you address them whenever
